web/src/main/webapp/xslt/common/render-html.xsl

```diff
@@ -74,6 +74,42 @@
           </div>
           <xsl:call-template name="footer"/>
         </div>
+
+        <!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
+        <script src="http://code.jquery.com/jquery-1.12.4.min.js"
```

Comment:
I don't think we should load extra libs in here. If we need JS in a formatter, then we can use the display service which load the JS and then load the formatter eg. http://localhost:8080/geonetwork/srv/fre/display#/435c454c-0d4b-41cf-a136-a1aba134d9ac/formatters/xsl-view?root=div

Reply:
Hi @fxprunayre, let me explain a bit why we've chosen this approach.

This view is targeting search engines and incidental users (arriving from search engines). 
In both cases we don't want to load the full geonetwork angular application, and most importantly not using ajax to load content (search engines will not parse that content).

We've chosen to import a minimal set of libraries to be able to make minimal layout optimisations (switching tabs), which degrade nicely for search engines and no-js users.

Sure we can use the jquery included in geonetwork, but using one from a common CDN may even speed up the experience for the intended audience

Reply:
The formatter view contains a link to the main app so I'm not 100% sure we should also have JS to maintain in here. 

![image](https://user-images.githubusercontent.com/1701393/33664863-5c08c78e-da95-11e7-9728-eaef82fb9c79.png)

But I'm fine with it if you think it is better.

BTW, when embedding the formatter in GN, you may have 2 versions of JQuery loaded which is probably not a good idea ?

Reply:
The JS is only loaded when `?root=html`, it is not loaded when `?root=div`.

Reply:
> The formatter view contains a link to the main app so I'm not 100% sure we should also have JS to maintain in here.

Jquery is required to make the tabs work on the html view of the page. I like the idea to have a link to the same content on the angular part of the portal